code/modules/clothing/head/misc.dm

```diff
@@ -259,4 +259,15 @@
 	name = "paper sack hat"
 	desc = "A paper sack with crude holes cut out for eyes and a sketchy smile drawn on the front. Not creepy at all."
 	icon_state = "papersack_smile"
-	flags_inv = HIDEHAIR|HIDEFACE|HIDEEARS
\ No newline at end of file
+	flags_inv = HIDEHAIR|HIDEFACE|HIDEEARS
+
+/obj/item/clothing/head/crown
+	name = "crown"
+	desc = "A crown fit for a king, a petty king maybe."
+	icon_state = "crown"
+	armor = list(melee = 50, bullet = 0, laser = 0,energy = 20, bomb = 0, bio = 0, rad = 0) //for a warrior king //energy cause gold is reflective //worse than sec helm overall

```

Comment:
50 melee resist on a stupid item

This is a new low.

Remove this please, because it makes it easier for steelpoint to justify his security armour buffs.